apps/admin-ui/src/common/Error5xx.tsx

```diff
@@ -3,7 +3,8 @@ import React from "react";
 import styled from "styled-components";
 import { H1 } from "common/src/common/typography";
 import { breakpoints } from "common/src/common/style";
-import { PUBLIC_URL } from "./const";
+import { FEEDBACK_URL, PUBLIC_URL } from "./const";
```

Comment:
This also should be prop drilled from PageWrapper

Reply:
Indeed.. fixed.